R/timeSinceFire.R

```diff
@@ -39,25 +42,19 @@ timeSinceFireUI <- function(id, rastersNumber) {
 #' @author Mateusz Wyszynski
 #'
 #' @export
-timeSinceFire <- function(input, output, session, rasters) {
+timeSinceFire <- function(input, output, session, rasters, polygons) {
```

Comment:
`polygons` is actually a list which elements are themselves sets of polygons, so maybe it would be better to rename this argument to something like `setsWithPolygons`

Reply:
what about naming it `polygonsList` so it's clear what it is from the name?

Reply:
`polygonsList` is still ambiguous to me. It might be as well interpreted as a list of polygons, i.e. each element is a polygon. Maybe `polygonsSetsList`?